### ISSUE TITLE
`getLastCallId` crashes on iOS

### ISSUE THREAD

**Issue by JulianBissekkou** [ConnectyCube/connectycube-flutter-call-kit#81]

**Comment by TatankaConCube** [comment]
Could you provide any info which will help us to localize the problem? Please add the crash log and/or info about the issue.

**Comment by JulianBissekkou** [comment]
I will send you a PR with all informations in a few hours 👍🏽

**Comment by JulianBissekkou** [comment]
```
[SwiftConnectycubeFlutterCallKitPlugin][handle] method: getLastCallId
connectycube_flutter_call_kit/SwiftConnectycubeFlutterCallKitPlugin.swift:70: Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

**Comment by tigrenok00** [comment]
Have the same issue including the crash. Can you fix it?

**Comment by JulianBissekkou** [comment]
I closed my PR but the fix is easy to do. I think @TatankaConCube planned to fix the issue in another PR. Maybe check with him 👍🏽

**Comment by TatankaConCube** [comment]
sorry but at this moment I have a lot of other tasks with higher priority and don't have time to review the PR and separate the desired changes

**Comment by JulianBissekkou** [comment]
@tigrenok00 
If you want to fix it by yourself you have to cherry pick the following commit: 1343dd79119109d1e08863cf3e2f9d2e7dece476
that should be it 👍🏽

**Comment by TatankaConCube** [comment]
will try to spend some time next week to fix this

**Comment by Merculiar** [comment]
I have the same issue, @TatankaConCube fix it, please.

**Comment by JulianBissekkou** [comment]
Guys, you just have to cherry pick this single commit. This will fix the issue. 
This will take less than 15min 😓

**Comment by TatankaConCube** [comment]
@Merculiar @booooohdan @tigrenok00 fixed in the version [2.2.2](https://pub.dev/packages/connectycube_flutter_call_kit/versions/2.2.2), please check it

**Comment by booooohdan** [comment]
@TatankaConCube 
Yes, not crashed anymore, thanks. 
You may close this ticket
